chatbot-ui/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 1.0.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.0.0"
+appVersion: "v0.2.3"
```

Comment:
Hmm I wonder if we should just make this part of the workflow so it auto tags for us:

https://github.com/redhat-composer-ai/appOfApps/blob/main/.github/workflows/release-tag.yml#L27